### DISCUSSION TITLE
Cruiser on social networks

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#689] [Show and tell]
Cruiser announcements will now also be on **X** (formerly **Twitter**). You can follow at this address:

[![X (formerly Twitter) Follow](https://img.shields.io/twitter/follow/devemux86?style=social)](https://twitter.com/devemux86)

The address is also on the main page.

**Comment by devemux86** [comment]
Please let me know if you would like to see Cruiser announcements on more social networks.

**Reply by bios4** [reply]
I think it all depends on whether you see added value in publishing on other social media platforms.
I personally use absolutely none of all of these services... 😜

EDIT: Therfore I also can't access the linked X-page (as it redirects me to log in).

**Reply by Maverikk65** [reply]
Hello Emux, it is of course one way of getting attention and reach out to get more customers... Personally I am on X (Twitter) extremely seldom (although I have an account), but I was watching a lot of funny movies on Instagram (and Facebook) lately... In order to get customers there you just have to have a good marketing strategy, then you might get some attention with relatively little effort... Good luck!

**Reply by DolomitesTom** [reply]
My use of social networks is 0%. I'm probably the last of the Mohicans.

**Reply by GuG1050** [reply]
Except WhatsApp I do not use any social networks. I would rather suggest to try something that makes it easier to find Cruiser when searching on Google. I have to admit I don't know how. But searching for a navigation system or whatsoever never results in finding Cruiser unless you know that it exists. At least in Germany.

**Reply by MotoUKRider** [reply]
I use [Instagram](https://www.instagram.com/thenoddingbiker/) to post ride pics and such, and have started using Threads for more "wordy" stuff but, with several other "micro blogging" outlets popping up, good engaging content is spread very thin.

Facebook would be the place for a popularity contest, although I would expect many messages/requests along the lines of: *"I want a pony"* especially if you want to compete in some way with another App. From what I see most bikers like the Calimoto/MyRouteApp approach for the integration and if you can't (or won't) compete with that then it's a non-starter unless there's a unique selling point of Cruiser.

So, who are you trying to attract and what are you offering?  That should help you decide where to put yourself socially.

I've never had much luck with searching for Cruiser under "motorcycle navigation" in GooglePlay. It's either eclipsed by the "bigger" Apps through SEO or simply the user-count.  I regard GitHub (and a sub-page Wiki) as fairly niche corners of the internet, too.

But then, I found it (during its K v2 era) and still use it!

**Reply by devemux86** [reply]
@MotoUKRider Very nice Instagram! :+1:
